src/index.html

```diff
@@ -16,21 +16,6 @@
         <h3>Recent Scores</h3>
         <div class="mobile">
           <ul class="scores" id="scores">
-            <li>
-              <h4>Shayan</h4>
-              <h4>-</h4>
-              <h4>900</h4>
-            </li>
-            <li>
-              <h4>Shayan</h4>
-              <h4>-</h4>
-              <h4>900</h4>
-            </li>
-            <li>
-              <h4>Shayan</h4>
-              <h4>-</h4>
-              <h4>900</h4>
-            </li>
           </ul>
           <button id="refresh">Refresh</button>
```

Comment:
Good job with auto-refresh after adding a new score but as the requirement says when you click refresh button it needs to refresh scores. Currently, there can be other people adding their scores, so you will not be updated until you refresh the page. The point of this project is to make async calls, so this is important. 👍 
<img width="1107" alt="Screenshot 2023-03-08 at 11 16 24 PM" src="https://user-images.githubusercontent.com/25887253/223841399-e283ae28-f385-47e5-a3e0-4054e7c57f83.png">